src/site/markdown/server/manage-webadmin.md

```diff
@@ -1548,7 +1548,22 @@ Response codes:
 
 ## Address Mappings
 
+You can use **webadmin** to define address mappings.
+
+When a specific email is sent to the base mail address, every destination addresses will receive it.
+
+This feature uses [Recipients rewrite table](/server/config-recipientrewritetable.html) and requires
+the [RecipientRewriteTable mailet](https://github.com/apache/james-project/blob/master/server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTable.java)
```

Comment:
do not use `master`, moving the file will break the documentation

Reply:
could you tell me what should be used instead of master because I saw other parts of the documentation also use master?